src/uTensor/ops/Convolution.hpp

```diff
@@ -46,10 +46,10 @@ class MaxFilter {
     tmp = std::max(tmp, input_value);
   }
   inline T finalize() const { return tmp; }
-  inline const int16_t height() const { return h; }
-  inline const int16_t width() const { return w; }
-  inline const int16_t in_channels() const { return 1; }
-  inline const int16_t out_channels() const { return c; }
+  inline int16_t height() const { return h; }
```

Comment:
Add const back if you want performance

Reply:
These are shouting with compilation warnings. Got a way to silence it in the Arduino IDE?

Reply:
no idea, but if you remove const here then the compiler cant optimize properly